Release_Notes.md

```diff
@@ -14,11 +14,11 @@ See the License for the specific language governing permissions and<BR>
 limitations under the License.*_<BR>
 
 # **Release Notes** #
-_*Last Update: 2023/12/08*_
+_*Last Update: 2023/01/17*_
```

Comment:
2024 :)

Reply:
Indeed, happy new year!
The fix for this was merged: https://github.com/chipsalliance/caliptra-rtl/pull/387/files